docs/src/references/child-devices.md

```diff
@@ -0,0 +1,120 @@
+
+On the other hand a child-device (external or logical) can announce it's capabilities to thin-edge to provoke thin-edge creating the cloud's child-device twin and declaring all capabilities on that child-device twin. 
+
+##### Announcing child-device capabilities by the child-device it-self
+
+To announce it's capabilities the child device (external or logical) sends an MQTT message to `tedge/meta/<childid>`. The MQTT message is as below:
+
+```json
+{
+   "device-name": "<optional name of that child-device>",
+   "device-type": "<optional type of that child-device>",
+   "<capability 1>": <capability specific JSON object>,
+   "<capability 2>": <capability specific JSON object>,
+   // [...]
+}
+```
+
+Thereby the fields are as below:
+   * `device-name` is an optional human readble device-name, visible in the cloud. If the field is not contained in the JSON message the `childid` of the topic structure is used as device-name.
+   * `device-type` is an optional device-type string assigned to the cloud's child-device twin. If the field is not contained in the JSON message the value `thin-edge.io-child` is used as device-type string.
+   * each field `<capability i>` represents a capability (e.g. `configurations` for _Configuration management_). The format of each `capability specific JSON object` is specific to the coresponding capability. For details see the documentation for coresponding capabilitie's API (e.g. [Configuration files for child devices](../references/c8y-configuration-management.md#configuration-files-for-child-devices)). 
+     Section [thin-egde APIs provided on network](#2-thin-egde-apis-provided-on-network-and-3-associate-consumedprovided-to-child-device-twins) below lists all APIs provided by thin-edge for child-device support.
+
+Example:
+```json
+{
+   "device-name": "My Child-Device",
+   "device-type": "type1",
+   "configurations": [ // capabilities for feature Configuration Management 
+     {
+       "type": "foo.conf"
+     },
+     {
+       "type": "bar.conf"
+     }
+   ]
+}
+```
+
+Whenever that MQTT message is sent to the thin-edge device:
+  1) thin-edge assures the child-device twin exists, and creates it if it does not exist.
+  2) each software component that provides an included capability (e.g. the C8Y Configuration Plugin for `configurations`) takes care to define all necessary capabilities to the coresponding cloud's child-device twin (i.E. _supported operations_, and list of provided types as e.g. _configuration types_).
+
+As result of the child-device creation thin-edge publishs an empty to MQTT topic `tedge/meta/success/<childid>` on successs, or a message to `tedge/meta/failed/<childid>`. The message on `tedge/meta/failed/<childid>` can optional contain a fail reaon as plain text.
```

Comment:
Need to document how we handle these failure cases:

1. If thin-edge is not configured to connect to Cumulocity when this message is received, will we do the provisioning after the connect happens?
2. When the device is connected, how do we handle a temporary network blip between the device and Cumulocity 
when this message is received?

Reply:
Removed from that PR, but to be considered in PR "requirements and use-cases" (#1195).
See more details in comment https://github.com/thin-edge/thin-edge.io/pull/1236#discussion_r955257236